scripts/test-size.js

```diff
@@ -15,42 +14,23 @@ try {
   process.exit(1)
 }
 
-// The last recorded size for these assets, as reported by `yarn build`.
-const LAST_SIZE_MAIN_KB = 435
-
-// This is the async-loaded js, called <number>.<hash>.js, with a matching css file.
-const LAST_SIZE_ENTRY_KB = 1442
-
+// The last recorded size for main.<hash>.js, as reported by `yarn build`.
+const SIZE_MAIN_KB = 1790
```

Comment:
What causes the chunk size to grow from 435 to 1790? Also, that is in KB, previously you mentioned `~7MB` in craco.config.js, so I am confused - what is the correct size here?

Reply:
https://github.com/Uniswap/interface/pull/6459#discussion_r1180898182